app/main.py

```diff
@@ -1,2 +1,52 @@
+import math
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.hash_table = [[key, None] for key in range(8)]
+        self.data = []
+        self.capacity = 0
+
+    def __setitem__(self, key: Any, value: Any) -> None:
+        self.data.append([key, value])
+        self.create_hash_table(key, value)
+        self.capacity += 1
+        if self.capacity >= math.floor(len(self.hash_table) * (2 / 3)):
```

Comment:
use constant value `2/3` it  is a common threshold used in hash table resizing strategies.

Reply:
But i`m using 2/3.What`s the point?

Reply:
I mean you should write it as constant. Constants are used to represent fixed values in a program that should not be changed during its execution.